lib/content-services/src/lib/aspect-list/aspect-list.component.ts

```diff
@@ -109,4 +105,11 @@ export class AspectListComponent implements OnInit, OnDestroy {
     getTitle(aspect: any): string {
         return aspect?.entry?.title ? aspect?.entry?.title : aspect?.entry?.id;
     }
+
+    hasEqualAspect(): boolean {
```

Comment:
Can we  use .includes insted of .find and have something like this:

`return this.nodeAspectStatus?.length !== this.nodeAspects.length ? false : this.nodeAspectStatus.every((aspect) => this.nodeAspects.includes(element);`

Even better wrap the last part into a function

`return this.nodeAspectStatus?.length !== this.nodeAspects.length ? false : isEveryAspectEtcEtc()`


Also can we rename this method to hasModifiedAspects, hasUpdatedAspects or something similar?